src/query/queryWithCursor.ts

```diff
@@ -16,9 +16,11 @@ export async function queryWithCursor<T extends AnyObject>(
   dbClient: DocumentClient,
   table: TableConfig,
   filter: Filter<T>,
-  indexName = 'default',
```

Comment:
This was only taking default index when not passed in argument right?

Reply:
yeah, but it also makes the query sending to DynamoDB to contain `IndexName = 'default'`, which is invalid

Reply:
this is where it goes wrong:
https://github.com/oolio-group/dynamo-helper/blob/bf110c17e8568e9f38dbe4b5414eb4af66e04442/src/query/queryWithCursor.ts#L42-L44